src/app/services/GetAdvertismentDetails/get-advertisment-details.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GetAdvertismentDetailsService {
+
+  private AD_DETAILS_REST_API_SERVER = "https://demo3944353.mockable.io/get-advertisment-details";
+
+  constructor(private httpClient: HttpClient) { }
+
+  public sendGetRequest() {
```

Comment:
Give name that describes what this function is doing.

Reply:
Done